consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/jsonrpc/IbftJsonRpcMethods.java

```diff
@@ -70,8 +71,9 @@ protected RpcApi getApiGroup() {
 
   private VoteTallyCache createVoteTallyCache(
       final ProtocolContext context, final MutableBlockchain blockchain) {
-    final EpochManager epochManager = context.getConsensusState(BftContext.class).getEpochManager();
-    final BftBlockInterface bftBlockInterface = new BftBlockInterface();
+    final BftContext bftContext = context.getConsensusState(BftContext.class);
```

Comment:
not sure why we pass in the blockchain to this function, given we extract everything else from the protocolcontext - why not just extract the blockchain too?

Reply:
done